lang/cs/Org.Apache.REEF.Client/YARN/RESTClient/DataModel/ApplicationState.cs

```diff
@@ -35,6 +35,10 @@ internal enum State
 
         RUNNING,
 
+        UNDEFINED,
+
+        SUCCEEDED,
+
         FINISHED,
```

Comment:
This enum does not have these values according to the documentation. Why are we adding these?
http://hadoop.apache.org/docs/r2.6.1/hadoop-yarn/hadoop-yarn-site/ResourceManagerRest.html

State 	The application state according to the ResourceManager - valid values are members of the YarnApplicationState enum: NEW, NEW_SAVING, SUBMITTED, ACCEPTED, RUNNING, FINISHED, FAILED, KILLED

Also see: https://github.com/apache/hadoop/blob/trunk/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/api/records/YarnApplicationState.java